client/login/magic-login/emailed-login-link-expired.jsx

```diff
@@ -37,11 +37,9 @@ class EmailedLoginLinkExpired extends React.Component {
 					replaceCurrentLocation={ true }
 				/>
 				<EmptyContent
-					action={ translate( 'Return to WordPress.com' ) }
-					actionCallback={ function() {
-						page( '/' );
-					} }
-					actionURL={ '/' }
+					action={ translate( 'Back to WordPress.com' ) }
+					actionCallback={ goBackToWordPressDotCom }
```

Comment:
@bisko -- That's what this was originally doing, just linking to `/` via the `actionURL` prop.

Check out the description of the problem this is solving (in this PR & in #15735).  Some routing magic somewhere was intercepting the page meaning that the route was changing to `/`, but the app state was not changing.

I think this is due to us not having a "real" root section in Calypso.  This should work around it.